src/store/result/reducers/result.reducer.ts

```diff
@@ -3,12 +3,13 @@ import { fetchPetsAsync } from '../actions/result.actions';
 import { StoreData } from './reducers.interfaces';
 import { PetProfile } from 'shared/interfaces/pet-profile.interface';
 
-const initiatState = {
+export const initiatState = {
   resultStore: [],
   errors: '',
 };
 
 export const resultReducer = createReducer(initiatState)
+  .handleAction(fetchPetsAsync.request, (state: StoreData) => ({ ...state }))
```

Comment:
Just advice: you may use additional property loading in your state and set it to true when you request data. It's quite comfy when you work with a loader/skeleton in your component.

Reply:
Thank you! Nice to know it ))